new_course/1_programming_fundementals/conditional_logic.md

```diff
@@ -83,135 +81,135 @@ end
 
 Ain't nobody pillaging my land!
 
-You can have as many elsif expressions as you want. The else clause is optional but you usually want to provide some default value in case none of the previous expressions evaluate to true.
+You can have as many `elsif` expressions as you want. The `else` clause is optional, but you usually want to provide some default value in case none of the previous expressions evaluate to `true`.
 
-## Boolean Logic
-To use an expression that evaluates to true or false you'll need a comparison operator. There are several provided by Ruby and in no particular order they are:
+#### Boolean Logic
+To determine whether an expression evaluates to `true` or `false`, you'll need a comparison operator. There are several provided by Ruby:
 
-`==`(equals) returns true if the values compared are equal.
+`==`(equals) returns `true` if the values compared are equal.
 ```ruby
-5 == 5 #=> true
-5 == 6 #=> false
+5 == 5 # => true
```

Comment:
Can we keep this as `#=>` for consistency with other lessons? I believe it's also convention to have it unspaced, but I could be wrong. @KevinMulhern?

Reply:
@leosoaivan Thanks for looking over these edits! 

Before I made these changes, I did a Google search to see if I could find this symbol used as a convention elsewhere, but I didn't find anything. Perhaps you can point me to where this is documented. 

I made this change for two reasons:
1. According to [multiple](https://www.digitalocean.com/community/tutorials/how-to-use-comments-in-ruby) [Ruby style guides](https://github.com/rubocop-hq/ruby-style-guide#hash-space), the appropriate convention is to add a space after the comment hash. 
2. Without this space, I felt it could be confusing to new students who might try to read "#=>" as all one symbol, which it's not. By adding the space, it makes it visually clearer that the "#" starts a comment and that the "=>" is just an annotation. 

In terms of consistency with other lessons, I plan to edit all of the lessons, and this change is easy enough to make throughout depending on the consensus here.

Reply:
Awesome! Thanks for pointing that out. I love me some sweet, sweet documentation. I'll add another commit to undo those spaces.